pkg/gadgets/trace/exec/tracer/tracer.go

```diff
@@ -156,6 +156,11 @@ func (t *Tracer) install() error {
 		return fmt.Errorf("attaching enter tracepoint: %w", err)
 	}
 
+	t.enterLink, err = link.Tracepoint("sched", "sched_process_exec", t.objs.IgSchedExec, nil)
+	if err != nil {
+		return fmt.Errorf("attaching sched_process_exec tracepoint: %w", err)
+	}
+
```

Comment:
This one took me a while. The tracer stopped working for me after ~2 minutes. After some investigation (blaming every single component you can imagine, from the terminal to the kernel), I found that this line is replacing the link on L154, causing the link for sys_enter_execve to be garbage collected. There should be a field for this one.

Reply:
Well spotted. Sorry for the mistake!